net-libs/libmicrohttpd/logs/custom-scanners-service.log

```diff
@@ -0,0 +1,84 @@
+2021-11-10 02:29:16.084  INFO 286042 --- [main] c.h.a.c.CustomScannersServiceApplication : Starting CustomScannersServiceApplication on fallenvalkyrie with PID 286042 (/home/arkaprabha/.vscode/extensions/hcltechnologies.hclappscancodesweep-1.1.4/lib/custom-scanners-service-1.0.11.jar started by arkaprabha in /home/arkaprabha/flatcar-sdk/src/third_party/portage-stable/net-libs/libmicrohttpd)
```

Comment:
When you move the PR to coreos-overlay, can you please delete `logs/custom-scanners-service.log`?
I suppose, that file is not needed.

Reply:
This one isn't supposed to be moved to coreos-overlay. This package is in portage. That was net-libs/rpcsvc-proto.

Reply:
But if you want I can remove this file.